package.json

```diff
@@ -14,9 +14,11 @@
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
     "@types/react-router-dom": "^5.1.7",
+    "axios": "^0.21.1",
```

Comment:
Do we need axios ?

Reply:
No, now we can use posts.json to get blog details. Removed this plugin